### DISCUSSION TITLE
What is the purpose of the ACTIVATE_HOOKS symbol?

### DISCUSSION THREAD

**Discussion by jamesvillarrubia** [feathersjs/feathers#2561] [Q&A] [answered]
I've been stepping through feathers v4 in order to build a new plugin with backwards compatibility for v4 and v5 and I have come across an odd Symbol that I'm not sure the purpose of:

https://github.com/feathersjs/feathers/blob/b5e94c4e996f64ef926b7d18849c854455d82f29/packages/feathers/lib/hooks/index.js#L137-L152

No matter the library or DB adapter, it seems like `service[key][ACTIVATE_HOOKS]` is always undefined, so this service.methods filter is always `[]` and the resulting list of methods is assigned to the default list.

Can someone explain what the intent of that symbol is?

**Comment by daffl** [comment, marked as answer]
It provides functionality to enable hooks on custom service methods that has been added but never been documented. In v5 this has been superseded by the general purpose https://github.com/feathersjs/hooks and the new [custom method system](https://dove.docs.feathersjs.com/api/services.html#custom-methods)

**Reply by jamesvillarrubia** [reply]
Got it!  I thought that might be the case.  I'm working with custom methods and they are getting filtered in v4, so I was investigating.  Thanks!
